### ISSUE TITLE
User block/unclock sample project

### ISSUE THREAD

**Issue by Shahjahan786** [QuickBlox/quickblox-android-sdk#585]
Kindly upload User block/unclock sample project like other samples, because this process is too confusing

**Comment by RomanPronin** [comment]
Hello @Shahjahan786, thanks for the suggestion, we will consider it.

**Comment by Shahjahan786** [comment]
Thanks @RomanPronin for quick response, actually I'm using enterprise QB account, we have trouble in block/unblock user, we have tried a lot but I stuck with this functionality. we have to add this functionality in our app same like WhatsApp do. we will be very thankful if you upload sample or java ready to use methods for block/unblock so we just call them in app and it work as expected

**Comment by kishorejethava** [comment]
Indeed required!!

**Comment by QB-nikolay-zolotarov** [comment]
Hello QuickBlox customer,

This is Nikolay from QuickBlox support.

The issue was closed as it was outdated.

Please check the relevant section of our documentation here:
[https://docs.quickblox.com/docs/android-chat-privacy-list](https://docs.quickblox.com/docs/android-chat-privacy-list)

Also, please update the SDK to the latest version:
[https://github.com/QuickBlox/quickblox-android-sdk-releases/releases/tag/3.9.11](https://github.com/QuickBlox/quickblox-android-sdk-releases/releases/tag/3.9.11)

If it is still relevant after reviewing the updated information, feel free to open a new issue.

Have a nice day.
